requirements_for_test.txt

```diff
@@ -1,10 +1,9 @@
 -r requirements.txt
-phonenumbers==8.10.13
-pytest==5.1.2
-pytest-mock==1.10.4
-pytest-cov==2.7.1
-pytest-xdist==1.29.0
-requests-mock==1.7.0
-freezegun==0.3.12
-flake8==3.7.8
-flake8-print==3.1.0
+pytest==5.4.3
```

Comment:
is phonenumbers not needed for tests?

Reply:
It's already listed in `setup.py` so I removed it here. Sorry, I should have added a note!